quarto/table-templates/template-table_08.qmd

```diff
@@ -0,0 +1,50 @@
+---
+title: FDA Table 8
+subtitle: All Individual Patient Deaths, Safety Population, Pooled Analyses
+format: html
+---
+
+::: panel-tabset
+## Spec. Screenshot
+
+![](../assets/images/screenshots/table_08.png){fig-align="center"}
```

Comment:
I would shrink the screenshot a bit since it's very wide
```suggestion
![](../assets/images/screenshots/table_08.png){fig-align="center" width="75%"}
```